src/pages/EditorPage.js

```diff
@@ -78,7 +80,7 @@ export default function EditorPage(props) {
   const [layout, setLayoutState] = useState(
     ls.get(EditorLayoutKey) || Layout.Split
   );
-  const [previewKey, setPreviewKey] = useState("");
```

Comment:
@jackson-harris-iii are all of these prettier reformats intentional?

Reply:
@charleslavon These prettier changes are not intentional. It's just how my editor is set. I'll adjust it for PRs moving forward.

Reply:
You can avoid adding these unintentional changes, and help other contributors, by including a `.prettierrc` config in this PR.  something like ```{
  "singleQuote": true,
  "trailingComma": "all",
  "singleAttributePerLine": false,
  "printWidth": 120
}```